### DISCUSSION TITLE
Grid export results in error when inheritance is disabled

### DISCUSSION THREAD

**Discussion by khenow1978** [pimcore/pimcore#16200] [General]
Hi.. in Pimcore there's something that bugs me (no pun intended) since our update to v11.0.12. 

If you try to export any grid view to xlsx with disabled inheritance it results in this error:

<img width="436" alt="image" src="https://github.com/pimcore/pimcore/assets/92732594/0c56814e-92bc-4318-8f38-ebfe41a12fdd">

It looks as if the value for the "GetInheritedValue" function isn't populated properly. Unfortunately I'm not a programmer and can't resolve it by myself.. is anybody able to reproduce this error?

**Comment by dvesh3** [comment]
I can confirm this is a reproducible bug. Reported https://github.com/pimcore/pimcore/issues/16202 thanks!
